mev_inspect/swaps.py

```diff
@@ -63,7 +64,7 @@ def _get_swaps_for_transaction(traces: List[ClassifiedTrace]) -> List[Swap]:
 
 
 def _parse_swap(
-    trace: ClassifiedTrace,
+    trace,
```

Comment:
no types?

If this creates an issue with mypy because there are potentially two types getting passed in you can use the `Union` type

`Union[ClassifiedTrace, ClassifiedTraceStrict]`